packages/opc-feedback/src/opc-feedback.scss

```
@@ -0,0 +1,108 @@
+@import '@one-platform/opc-styles';
+
+$LOCAL: feedback;
+
+$variables: ( 
+  //Add your variables here
+);
+
+:host {
+  --op-transition--default: 120ms ease-in-out;
+  --op-feedback__btn-primary: #b70202;
+  --op-feedback__bg-primary: #fceaeb;
+
+  position: relative;
+  z-index: 9000;
+}
+
+* {
+  outline-width: medium;
+  font-family: 'Red Hat Text', 'Open Sans', Arial, Helvetica, sans-serif;
+}
+
+.op-feedback__button {
+  position: fixed;
+  bottom: 1.5rem;
+  right: 1.5rem;
+  display: flex;
+  flex-direction: row;
+  align-items: center;
+  background: opc-get-var(BackgroundColor, $opc-color--red-200);
+  color: opc-get-var(TextColor, $opc-color--white);
+  border: none;
+  line-height: 2rem;
+  padding: .5rem 1.25rem;
+  font-size: 1rem;
+  cursor: pointer;
+  border-radius: 2rem;
+  border-bottom-right-radius: .25rem;
+  box-shadow: 0 .25rem .5rem rgba(0,0,0,.16);
+  transition: all var(--op-transition--default);
+  &:hover, &:focus {
+    background: opc-get-var(BackgroundColor, $opc-color--red-300);
+    box-shadow: 0 .5rem .5rem rgba(0,0,0,.24);
+  }
+}
+
+.op-feedback__panel {
```

@riginoommen Same as above
```suggestion
.op-feedback {
&__panel {
```

---

Applied the suggested changes